requirements/base.txt

```diff
@@ -0,0 +1,13 @@
+#
```

Comment:
Could you please update the Dockerfile for the sample project? It's breaking now since the requirements filename was renamed:

![image](https://user-images.githubusercontent.com/9268203/94952131-dc848580-04bb-11eb-8b5b-bf7cbc910ab3.png)

Reply:
🤦 my bad

Reply:
It is not in the sample-project folder, but the celerylogger Dockerfile